packages/botonic-plugin-contentful/src/cms/cms-error.ts

```diff
@@ -111,14 +111,14 @@ export class ErrorReportingCMS implements CMS {
       .catch(this.handleError('contentsWithKeywords', context))
   }
 
-  topContents(
+  topContents<T extends TopContent>(
     model: TopContentType,
     context?: Context,
     filter?: (cf: CommonFields) => boolean,
     paging?: PagingOptions
-  ): Promise<TopContent[]> {
+  ): Promise<T[]> {
     return this.cms
-      .topContents(model, context, filter, paging)
+      .topContents<T>(model, context, filter, paging)
```

Comment:
Is it necessary to add here the `<T>`?

Reply:
yes, otherwise TS will complain that the function is declared as returning T[], but it returns TopContents[]